src/main/java/io/phasetwo/keycloak/magic/MagicLink.java

```diff
@@ -143,6 +163,7 @@ public static MagicLinkActionToken createActionToken(
       String scope,
       String nonce,
       String state,
+      String codeChallenge,
```

Comment:
@xgp I've noticed the codeChallenge parameter is not passed in to the `MagicLinkActionToken` constructor. If I pass it in and run it against my setup I get a different PKCE failure, `Code mismatch` instead of `challenge not present`.

Reply:
Thanks. Feel free to take up this PR.

Reply:
#91